x-pack/plugins/aiops/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prefix": "aiops",
+  "paths": {
+    "aiops": "."
+  },
+  "translations": ["translations/ja-JP.json"]
```

Comment:
Is this file needed? I don't see it in other plugins. If so, why does it only include `JP` ?

Reply:
Removed in [b393103](https://github.com/elastic/kibana/pull/131317/commits/b393103ab47fa6aa7e19c6c24f33e8e3b055b0ee).